### PR TITLE
Adding report deployment step template.

### PR DESCRIPTION
**Pull request by harbertc**
### Step template checklist

- [X] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [X] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [X] Parameter names should not start with `$`
- [X] **Step template parameter names (the ones declared in the JSON, not the script body) should be prefixed with a namespace so that they are less likely to clash with other user-defined variables in Octopus** (see [this issue](https://github.com/OctopusDeploy/Issues/issues/2126)). For example, use an abbreviated name of the step template or the category of the step template).
- [x] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [N/A] If a new `Category` has been created:
   - [N/A] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [N/A] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

### CODE REVIEW

**Comment by twerthi** [comment]
Thank you for your submission :)  Would you be able to add something that checks to make sure pip is installed and reports an error if it is not?

**Comment by harbertc** [comment]
@twerthi I added the following which should provide a nicer message if pip is not available. Please let me know if this is what you had in mind.

```
try {$pipCmd = get-command pip} catch {}
if (!($pipCmd)) {
	Fail-Step "This step template requires Python 3.6 or greater and pip to be installed. Python is available at https://www.python.org/downloads/"
}
```

**Review by harrisonmeister — CHANGES_REQUESTED** [review]
One minor change to revert the Id of `testery-create-test-run` and I think you are good to go!

File: step-templates/testery-create-test-run.json [hunk 1]
```diff
@@ -1,116 +1,85 @@
 {
-  "Id": "9aa25d78-a90a-425e-a25d-b1e9f1bf08be",
+  "Id": "ed901ef2-9799-4f29-90c0-e1a8e6d92a2c",
```
[the remark this answers is not in the document]
Comment (harbertc) [reply step-templates/testery-create-test-run.json]: @harrisonmeister - Done. Used a new instance of Octopus to generate the new step template and that was auto-generated. I've set it back.

**Review by harrisonmeister — APPROVED** [review]

**Pull request merged**